src/bin/laminar-tester.rs

```diff
@@ -99,7 +101,15 @@ fn run_server(socket_addr: &str) {
 
 fn run_client(test_name: &str, destination: &str, endpoint: &str, pps: &str, test_duration: &str) {
     let network_config = net::NetworkConfig::default();
-    let mut client = net::UdpSocket::bind(endpoint, network_config.clone()).unwrap();
+    let mut client = match net::UdpSocket::bind(endpoint, network_config.clone()) {
+        Ok(c) => { c }
+        Err(e) => { 
+            println!("Error binding was: {:?}", e);
+            exit(1);
+
```

Comment:
extra space here 😄 , do you have rustfmt setup on save? if not I really recommend it.

Reply:
I think we're covering that in our style guide discussion.